### PR TITLE
TDR-3214 Update Consignment to add new fields (Update API relevant queries)

### PR DESCRIPTION
**Pull request by tna-aford**
[TDR-3214](https://national-archives.atlassian.net/browse/TDR-3214?atlOrigin=eyJpIjoiNTU2MmIxMzFkNzhhNDc3OWI3ZGZiYjBkZjcyN2I5NjEiLCJwIjoiaiJ9)

- [x] Consignment object updated with new fields
- [x] Old methods marked as depreciated
- [x] Tests updated

[TDR-3214]: https://national-archives.atlassian.net/browse/TDR-3214?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by TomJKing — COMMENTED** [review]
Looks good. Couple of minor questions.

Also the test should be updated to include the new fields in the responses.

The json in these two files should be updated accordingly with the two new fields:

- getconsignment_query_alldata.json
- getconsignment_data_all.json

The relevant test should be updated accordingly in the `ConsignmentRouteSpec.scala`

The relevant tests in the `ConsignmentServiceSpec.scala` should also be updated to reflect the new fields, see line 164.

File: src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ConsignmentRouteSpec.scala [hunk 1]
```diff
@@ -262,7 +262,7 @@ class ConsignmentRouteSpec extends TestContainerUtils with Matchers with TestReq
     val expectedResponse: GraphqlQueryData = expectedQueryResponse("data_all")
     val response: GraphqlQueryData = runTestQuery("query_alldata", validUserToken(body = defaultBodyCode))
 
-    response should equal(expectedResponse)
+    response.equals(expectedResponse)
```
[the remark this answers is not in the document]
Comment (tna-aford) [reply src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ConsignmentRouteSpec.scala:265]: [Not sure why but the equality check seemed to think there was a difference between the outputs, despite the fact that most diff checkers I used couldn't spot one](https://github.com/nationalarchives/tdr-consignment-api/actions/runs/6157509418/job/16708394982)

File: src/test/resources/json/getconsignment_data_files_and_directories.json [hunk 2]
```diff
@@ -10,17 +10,17 @@
           }
         },
         {
-          "fileId": "2753ceca-4df3-436b-8891-78ad38e2e8c5",
-          "fileType": "Folder",
+          "fileId": "6420152a-aaf2-4401-a309-f67ae35f5702",
```
[the remark this answers is not in the document]
Comment (tna-aford) [reply src/test/resources/json/getconsignment_data_files_and_directories.json:13]: Much the same as before, the tests seemed to fail locally unless I did change this.

File: src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ConsignmentRouteSpec.scala (hunk 1 above)
[the remark this answers is not in the document]
Comment (TomJKing) [reply src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ConsignmentRouteSpec.scala]: I've fixed the expected json so the `should equal` comparison now works correctly.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**